toml.md

```diff
@@ -614,9 +629,10 @@ contributors = [
 ]
 ```
 
-Arrays can span multiple lines. A terminating comma (also called trailing comma)
-is permitted after the last value of the array. Any number of newlines and
-comments may precede values, commas, and the closing bracket.
+Arrays can span multiple lines. A terminating comma (also called a trailing
+comma) is permitted after the last value of the array. Any number of newlines
+and comments may precede values, commas, and the closing bracket. Indentation is
+treated as whitespace.
```

Comment:
> Indentation is treated as whitespace.

Will that be sufficient to eliminate confusion for the Python/YAML folk, given their relationship with "whitespace"? Suggest it include the word "ignored" somewhere.

Reply:
Good idea. I'll add "ignored" to the language here.

Reply:
Whoops, I missed your previous comment. But since we're introducing this clarification for pedantic reasons, the duplication language is essential. We could come back in the future and sort-of refactor, but not just yet.

Reply:
Roger that

(incidentally you saw my notes in the intended order; my inline comment was actually first and the general commentary on the duplication was the final "summary" comment at the bottom of the review, but github displays them like this after-the-fact, which is super weird)